assets/css/admin-custom.scss

```diff
@@ -162,10 +162,7 @@ div.question_boolean_fields { margin-bottom: 10px; }
  * Sensei custom navigation
  */
 .sensei-custom-navigation {
-	display: flex;
-	flex-flow: column wrap;
-	align-items: flex-start;
-	gap: 20px;
+	display: grid;
```

Comment:
My suggestion here would be to keep the previous code, because I think it would be better to move the notifications outside of the navigation. That way we won't have to account for them being there. See the next comments for more info.

Reply:
Forgot to mention that the grid view changes how the navigation looks on the other screen (some of the spacing is missing), but we won't have to worry about that if we decide to move the notifications outside.